test/mocks/MockFeatureSwitches.scala

```diff
@@ -29,6 +29,15 @@ trait MockFeatureSwitches extends MockFactory {
     def isPassDeleteIntentEnabled: CallHandler[Boolean] =
       (() => mockFeatureSwitches.isPassDeleteIntentEnabled).expects()
 
+    def isAllowNegativeExpensesEnabled: CallHandler[Boolean] =
+      (() => mockFeatureSwitches.isAllowNegativeExpensesEnabled).expects()
```

Comment:
I've added it in as it should be used - there are several places in this API that are using a slightly convoluted approach to feature switches but decided not to go through and update all those places yet as it's outside the scope of this ticket to amend those.  With these added in it should encourage the correct approach in the future

Reply:
I agree it might be a good idea to improve our approach to feature switches, but I think we should add this function in when we start updating the approach in other places too. Adding it in now seems to break the[ team development standard](https://confluence.tools.tax.service.gov.uk/display/MTE/Team+Development+standards) of "Don't write code that you think you might need in future."

Reply:
Have updated the PR which now uses this function